test/commands/scanner/rule/describe.test.ts

```diff
@@ -80,19 +80,19 @@ describe('scanner:rule:describe', () => {
 			// Both tests will test for the presence of this warning string in the output, so we might as well format it up here.
 			const formattedWarning = messages.output.multipleMatchingRules
 				.replace('{0}', '2')
-				.replace('{1}', 'WhileLoopsMustUseBraces');
+				.replace('{1}', 'constructor-super');
```

Comment:
Most of the previous tests would fail with errors from both eslint and PMD. I modified any tests where this duplication of results was important. I changed the expected results in other tests where the duplication wasn't important to the test.